src/main/java/zmaster587/advancedRocketry/dimension/DimensionManager.java

```diff
@@ -373,25 +374,14 @@ public DimensionProperties generateRandomGasGiant(int starId, String name, int b
 		properties.setGasGiant(true);
 
 		// Add gasses to gas giants, for Jupiter-like planets add hydrogen (and helium and helium-3 if they exist) and for ice giants add nitrogen and oxygen, with ammonia and methane if they exist
-		if(properties.getGravitationalMultiplier() > 1.25){
-			properties.getHarvestableGasses().add(AdvancedRocketryFluids.fluidHydrogen);
-			if (FluidRegistry.isFluidRegistered("helium")){
-				properties.getHarvestableGasses().add(FluidRegistry.getFluid("helium"));
+		for( Fluid gas : AdvancedRocketryFluids.getGasGiantGasses() ) {
```

Comment:
I feel kinda bad kicking this back again but there's an edgecase in this loop, gasgiants with a gravity multiplier of 1.25 will end up barren, we probably want to make sure we cover that case.

Honestly though, even though this new code is more compact, it's also less readable and might be harder to maintain going forward as new gasses are added.  A single if statement wouldn't scale well (readability-wise) I'd imagine.

Reply:
I can push up new code `isGasgiantGasRegistered(String name)` for convenience.